museek/util/time_analysis.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from datetime import datetime, timedelta
+import ephem
+from museek.time_ordered_data import TimeOrderedData
+
+class TimeAnalysis:
+    """ Class to do time-related analysis. """
+
+    def __init__(self, data: TimeOrderedData):
+        """
+        Initialise
+        :param data: the analysed `TimeOrderedData` object
+        """
+        self.obs_start = datetime.utcfromtimestamp(float(data.original_timestamps[0])) 
```

Comment:
This class can at the moment only be used together with the `TimeOrderedData` class. If you put lat and lon into the init instead of `data` it will be much more versatile.
obs_start and obs_end are specific parameters to `time_difference_to_sunset_sunrise` and they should best not be in the `init`

Reply:
updated